lib/screens/AppLoad/index.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:loading/loading.dart';
+import 'package:loading/indicator/line_scale_party_indicator.dart';
+
+
+class AppLoad extends StatefulWidget {
+
+  @override
+  _AppLoadPageState createState() => _AppLoadPageState();
+
+}
+
+class _AppLoadPageState extends State<AppLoad> {
+
+  Future<String> token;
+
+  Future<String> _startPage() async {
+    SharedPreferences prefs = await SharedPreferences.getInstance();
+    return Future.delayed(const Duration(seconds: 1,milliseconds: 500), () {
```

Comment:
Why the delay and why that value?

Reply:
If there are no delay, the loading screen will only flash, because the search is fast. But there has to be something there or else a white screen will flash between the splash screen and the next screen (home if logged in or login if not). The time was defined in terms of fluidity, in a manner that wasn't too short nor too long.